pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = ["setuptools >= 46.4.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+
+# This is kept for posterity; `black` is not used in this repository
+[tool.black]
+line-length = 120
```

Comment:
Might as well drop this if we're not even installing it? Or are we planning on using it later?

Reply:
@MJJoyce maybe we'll figure out how to keep Black from touching `.xml`, `.tab`, `.TAB`, files some day.

The fact it even rewrites `.rst` and `.md` files is pretty … extreme. I have a feeling there may be some weird interactions in the future between our automatically-generated `CHANGELOG.md` and Black 👽